### PR TITLE
feat: upgrade to dotnet 5.0

### PR DESCRIPTION
**Pull request by sjk07**

### CODE REVIEW

File: src/AM.Condo/Tasks/FindCommand.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class FindCommand : Task
         public string WorkingDirectory { get; set; }
 
         /// <summary>
-        /// Gets or sets the search paths to use when locating the command.
+        /// Gets the search paths to use when locating the command.
```
Comment (sjk07) [inline src/AM.Condo/Tasks/FindCommand.cs]: or sets

File: src/AM.Condo/Resources/NodeProject.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ public class NodeProject
         public string License { get; set; }
 
         /// <summary>
-        /// Gets or sets the scripts for the project.
+        /// Gets the scripts for the project.
```
Comment (sjk07) [inline src/AM.Condo/Resources/NodeProject.cs]: or sets

**Review by sjk07 — COMMENTED** [review]
reviewing my own PR so I don't forget

**Review by dmccaffery — DISMISSED** [review]
LGTM!

**Comment by sjk07** [comment]
CircleCI and appveyor will fail because the condo docker images do not contain dotnet v5 SDK

**Pull request merged**